react/Router.js

```diff
@@ -0,0 +1,12 @@
+import * as ReactRouterDom from 'react-router-dom'
+
+/*
+  IMPORTANT: READ BEFORE MAKING ANY EDITIONS IN THIS FILE
+
+  This file's goal is to make the use of react-router-dom
+  unified through all the apps within the My Account ecosystem.
+  Delete this if a better solution to the problem of different
```

Comment:
```suggestion
  Delete this if a better solution to the problem or different
```

Reply:
The ideia here is to refer to the problem's characteristics. Why do you think using 'or' would be a good fit here?

Reply:
ah, I thought it was a typo. nvm